app/networkapi/people/serializers.py

```diff
@@ -30,5 +30,5 @@ class Meta:
             'quote',
             'affiliations',
             'internet_health_issues',
-            'partnership_logo'
+            'partnership_logo',
```

Comment:
as for the why: python doesn't mind extra commas, but not having enough commas in certain listing syntax will completely change the meaning and do radically different things. trailing commas are, unfortunately, a good thing in Python